docs/connecting-to-a-lit-network/mainnets.md

```diff
@@ -0,0 +1,48 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Mainnets
+
+Main networks are designed for late-stage application deployment. If you are implementing storing assets with real world value, it should be done on the main networks and not the test networks. While main networks may be deprecated in the future, assets will be transferable onto new networks.
```

Comment:
> Main networks are designed for late-stage application deployment.

Consider, "Lit mainnets are designed for late-stage development and production deployment"

Reply:
> While main networks may be deprecated in the future, assets will be transferable onto new networks.

Consider "While a given mainnet may be deprecated in the future, all assets and key material generated on a Lit mainnet will be recoverable and useable on future network releases"